testnet/stacks-node/src/tests/integrations.rs

```diff
@@ -174,9 +189,29 @@ fn integration_test_get_info() {
 
             if round == 1 {
                 // block-height = 2
+                eprintln!("Tenure in 1 started!");
```

Comment:
I'm wondering if we should rethink how we are adding tests in `integrations.rs` in general. The test function `integration_test_get_info` is now over 600 lines long, and its original use seems to be to test the `GET_INFO_CONTRACT`. 

Maybe we should break tests for this change into its own function (or maybe not, because it might be time-consuming to run many individual integrations tests). Thoughts @jcnelson ?

Reply:
I can do this, but this will probably increase the overall testing time, given that you'd need to re-setup the same thing over and over.

Reply:
after thinking this further, this is something that could be considered separately, but is outside the scope of this PR